cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -376,6 +387,27 @@ func (c *Client) AddMember(name string, peerAddrs string) ([]Member, error) {
 		ctx, cancel := context.WithTimeout(context.Background(), etcdTimeout)
 		defer cancel()
 		var resp *clientv3.MemberAddResponse
+		var learnerID uint64
```

Comment:
should this be outside of the main exp backoff loop?

Reply:
Yes, my mistake.